robot.go

```diff
@@ -13,9 +13,12 @@ import (
 	"golang.org/x/sync/errgroup"
 	"golang.org/x/time/rate"
 
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
```

Comment:
These imports should be with the chunk above. The style I prefer is that each of stdlib, module-external packages, and module-internal packages are grouped separately, so that the location of the import reflects where to go to find it (stdlib, go.mod, or the repo itself).